packages/importer/src/index.ts

```diff
@@ -130,10 +135,11 @@ class OneSchemaImporter extends EventEmitter {
       )
     }
 
-    if (this.iframe.contentWindow) {
+    if (this.#isLoaded) {
```

Comment:
Found a scenario where `contentWindow` would exist, but things aren't loaded... this seems a bit more foolproof.. using onload to set a loaded flag and if isn't set, assume we haven't loaded/use the onload func for our init..